dev_tools/sample_request.py

```diff
@@ -0,0 +1,20 @@
+import requests
+import json
+
+url = "http://127.0.0.1:8080"
+r = requests.post(url+"/debug/login", data={"email": "test@test.de", "password": "1234"})
+
+
+token = r.json()["token"]
+headers = {'Accept': 'application/json', 'Authorization': str('JWT '+token)}
```

Comment:
JWT was simply choosen because the library used with the jwt tokens had this as the default value. According to https://jwt.io/introduction/ this should be changed to `Bearer`